ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml

```diff
@@ -70,6 +70,13 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-preserve-bootstrap-ign
+- as: gcp-ipi-proxy-workload-identity-f28
```

Comment:
We are placing this job here on purpose, only for installer team testing, but not involve the whole team's e2e testing, right?

Reply:
Actually I am thinking if we can place it in the common ci yaml files as ususal.

Reply:
In the common ci, there are tests like "**gcp-ipi-disconnected-private-workload-identity**", which covers the scenario of disconnected + workload-identity. It's very similar to this test, so I suggest to put it in installation dedicated test. So?

Reply:
if we think proxy + oidc is still needed to tested even we already have disconnecdted + oidc, I prefer to place it in common ci for a full coverage from the whole team view, and we can set a lower frequency when adding it to common ci.

Reply:
Got it, let me update, thanks!